src/current/v25.2/load-based-splitting.md

```diff
@@ -71,25 +71,42 @@ This benefit is further amplified by [load-based rebalancing]({% link {{ page.ve
 
 ## Monitor load-based splitting
 
+### Log message
+
 The [`INFO`]({% link {{ page.version.version }}/logging.md %}#info) log message
 
 ~~~
 no split key found: ...
 ~~~
 
-indicates that CockroachDB wants to [split the range]({% link {{ page.version.version }}/architecture/distribution-layer.md %}#range-splits) due to load, but couldn’t find a key to split at.
+indicates that CockroachDB attempted to [split the range]({% link {{ page.version.version }}/architecture/distribution-layer.md %}#range-splits) due to load, but could not find a key to split at.
+
+This log message can usually be ignored. However, if it appears repeatedly, it may indicate a load imbalance in the cluster. A load imbalance might occur if a [hot range]({% link {{ page.version.version }}/understand-hotspots.md %}#hot-range) cannot be split because it is effectively a [hot key]({% link {{ page.version.version }}/understand-hotspots.md %}#row-hotspot). For more information about how to reduce hotspots (including hot ranges) on your cluster, refer to [Understand hotspots]({% link {{ page.version.version }}/understand-hotspots.md %}).
```

Comment:
Can we make the following edits, because I think these sentences can be now genericised further?

A load imbalance might occur if a hot range... -> A load imbalance might occur if a **range**...

it is effectively a hot key... -> it is effectively a hotspot...